code/modules/spacepods/spacepod.dm

```diff
@@ -54,12 +68,38 @@
 	var/move_delay = 2
 	var/next_move = 0
 
+/obj/spacepod/proc/apply_paint(mob/user as mob)
+	var/part_type
+	var/part = input(user, "Choose part", null) as null|anything in list("Lights","Rim","Paint","Windows")
+	if(part == "Lights")
```

Comment:
This might sound dumb, but I'm not very familiar with DM. If I put the defines in the list, they'll come up as numbers when the user views them no?